src/main/kotlin/parts/wisdom/arcraider/SimpleLineWorld.kt

```diff
@@ -0,0 +1,42 @@
+package parts.wisdom.arcraider
+
+import me.joypri.Part
+import me.joypri.to
+
+/**
+ *
+ * This file is just an example of how to use our building blocks.
+ */
+
+
+val myGrid = Grid(
```

Comment:
Just FYI, this is where `Remix` is intended to come in: it provides a DSL for nested construction of clans. But what you have here is, in my way of seeing things, a fine alternative for cases where it's not worth the trouble to define corresponding `Remix`s.

Reply:
TBH I don't know if I fully understand what the purpose of the Remixes are. It seems like they would be used in cases where you need to construct a Mix, but don't have all the necessary data yet. So you create a Remix and fill in the rest of it at some later point and convert it to the corresponding Mix. Am I understanding that right? 
If so, then in this case, I do have all the data (because it's all hardcoded stuff), so conceptually why would it make sense to use a Remix?

Reply:
You have the essence of `Remix` exactly right. But there's a second use for `Remix`: it has built-in DSL support. (See the tests.) I don't know a good way for `Mix` to have built-in DSL support given that it is immutable.